src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import bookReducer from './books/books';
+import categoryReducer from './categories/categories';
+
+const allReducer = combineReducers({
+  bookReducer,
+});
```

Comment:
- [ ] Please include the `categoryReducer` in the `combineReducers` function here instead of putting it in the `configureStore` function directly.